node/builder_chain.go

```diff
@@ -48,9 +48,16 @@ var ChainNode = Options(
 	// Full node or lite node
 	// TODO: Fix offline mode
 
+	// FVM: builtin actor bundle loading
+	//      Note: this has to load before the upgrade schedule, so that we can patch in the
+	//      right manifest cid.
+	//      This restriction will be lifted once we have the final actors v8 bundle and we know
+	//      the manifest cid.
+	Override(new(dtypes.BuiltinActorsLoaded), modules.LoadBultinActors),
```

Comment:
We'll also need to load the manifest on the miner/market nodes as those are using `chain/actors` on top of API-backed blockstores.

This should probably be done as an API helper method, so that other Go programs importing lotus can do that easily too.

Reply:
ah, good catch.

Reply:
we'll have to discuss this in the huddle, not sure how to do it.